### DISCUSSION TITLE
[Feature Request] Global Actions View

### DISCUSSION THREAD

**Discussion by cablespaghetti** [community/community#18084] [Actions]
We are in the process of moving our organisation from a mixture of other CI/CD products to GitHub Actions. We love the product but one feature we really miss is the ability to see all builds currently running across our organisation. This is useful for seeing if there are a lot of queued or failing builds due to issues with our self-hosted runners or otherwise.

If there is an API we could use to have some kind of view of this that would work as a stopgap.

**Comment by mikeschinkel** [comment]
Hey @cablespaghetti  — I agree with your suggestion, my manager would really like that too.

However, unless and until GitHub offers such as feature how you seen https://meercode.io?  #justfyi #fwiw
